airflow/www/security.py

```diff
@@ -32,6 +31,7 @@
 from airflow.security import permissions
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.session import provide_session
+from airflow.www.fab_security.sqla.manager import SecurityManager
```

Comment:
Is the plan to customize and tweak the content of these files?

If not, It maybe makes more sens to put them under airflow/_vendor/

Or maybe even if we do plan on doing that, if the content is "as upstream" now, maybe it still makes sense to put in that location until we do make such a change.